Cargo.toml

```
@@ -10,15 +10,16 @@ dashmap = "5.4"
 enum_dispatch = "0.3"
 futures = "0.3"
 futures-util  = "0.3"
-greptime-proto = { git = "https://github.com/GreptimeTeam/greptime-proto.git", tag = "v0.4.1" }
+greptime-proto = { git = "https://github.com/GreptimeTeam/greptime-proto.git", tag = "v0.7.0" }
 parking_lot = "0.12"
 prost = "0.12"
 rand = "0.8"
 snafu = "0.7"
 tokio = { version = "1", features = ["rt", "time"] }
 tokio-stream = { version = "0.1", features = ["net"] }
-tonic = { version = "0.10", features = ["tls", "tls-roots"] }
+tonic = { version = "0.11", features = ["tls", "tls-roots", "gzip", "zstd"] }
 tower = "0.4"
+derive_builder = "0.20.0"
```

```suggestion
derive_builder = "0.20"
```

Remove the patch version.